tutorials/c7n-tutorial-cleanup.yml

```diff
@@ -0,0 +1,10 @@
+policies:
+  - name: my-first-policy-event-terminate-tagged-ec2
+    resource: aws.ec2
+    filters:
+      - tag:c7n-tutorial: present
+    actions:
+      - terminate
```

Comment:
Swap the terminate and tag actions as you can't tag a resource after it has started the termination process